documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -100,6 +100,11 @@ If required, set the `version` property for Kafka Connect and MirrorMaker as the
 .. For Kafka Connect, update `KafkaConnect.spec.version`.
 .. For MirrorMaker, update `KafkaMirrorMaker.spec.version`.
 .. For MirrorMaker 2, update `KafkaMirrorMaker2.spec.version`.
++
+NOTE: If you are using custom images that are built manually, you must rebuild those images to include the new version of the client binaries. 
+For example, if you xref:creating-new-image-from-base-str[created a Docker image from the base Kafka Connect image], update the Dockerfile to point to the latest base image and build configuration.
+
+. Verify that the upgraded client applications work correctly with the new Kafka brokers.
```

Comment:
This is not really the right thing to say. The client binaries are not really an issue as this suggests - updating them is not sufficient in any way. They have to use the new Strimzi image as the base image to have the latest _image_.